containers/arangodb35.docker/entrypoint.sh

```diff
@@ -1,6 +1,68 @@
 #!/bin/sh
 set -e
 
+## Update conf from generic env vars
+## env vars syntax:
+##  ARANGOCONF_SECTION_PARAMETER=VALUE
+## example:
+##  ARANGOCONF_LOG_LEVEL=debug
+##   => update [log] section with "level = debug"
+updateconf() {
+  tmpconfdir=/tmp/arangoconf
+  conffile=/tmp/arangod.conf
+
+  # first section has no name. Will be id 0, name "header"
+  currentfile=header
+  currentnumber=0
+
+  mkdir -p "$tmpconfdir"
+  rm "$tmpconfdir"/[0-9]* 2>/dev/null || true
+
+  ## Split config file per section
+  while read -r line; do
+      if echo "$line" | grep -q -e '^\[.*\]'; then
+          currentfile="$(echo "$line" | sed -n 's/.*\[\(.*\)\]/\1/p')"
+          currentnumber=$((currentnumber+1))
+      fi
+      echo "$line" >> "$(printf "%s/%02d-%s" "$tmpconfdir" "$currentnumber" "$currentfile")"
+  done < "$conffile"
+
+  ## cycle through all ARANGOCONF* var / value pairs
+  #  lowercase values while we're here because it's easy with awk
+  awk 'BEGIN{for(v in ENVIRON) print tolower(v) " " ENVIRON[v]}' \
+    | grep -e '^arangoconf' \
+    | while read -r var value; do
+
+        ## Extract section name and parameter name from var name.
+        # :: In POSIX sh, process substitution is undefined. In bash we would do:
+        # :: $ read -r section param < <(echo "$var" | sed -n 's/ARANGOCONF_\([^_]*\)_\(.*\)/\1 \2/p')
+        # :: so let's split it in 2 separate parsings
+        section=$(echo "$var" | sed -n 's/arangoconf_\([^_]*\)_\(.*\)/\1/p')
+        param=$(echo "$var" | sed -n 's/arangoconf_\([^_]*\)_\(.*\)/\2/p')
+        param=$(echo "$param" | sed 's/_/-/') # env vars use '-' but param names use '-'
```

Comment:
@hervenicol minor issue: can you update the comment the first '-' should be '_'

Reply:
Fixed (I managed to insert a typo in the commit message :man_facepalming: )

Reply:
that'll be squashed out - no worries..

Reply:
@fceller , could you please review the fix, and confirm the resolution if it's good for you ?